src/main/java/flaxbeard/steamcraft/block/BlockBoiler.java

```diff
@@ -198,7 +199,8 @@ public void registerBlockIcons(IIconRegister ir) {
         this.boilerOffIcon = ir.registerIcon("steamcraft:boiler");
         this.camoOnIcon = ir.registerIcon("steamcraft:boilerCamoOn");
         this.camoIcon = ir.registerIcon("steamcraft:boilerCamo");
-        //this.field_149936_O = p_149651_1_.registerIcon(this.field_149932_b ? "steamcraft:boilerOn" : "steamcraft:boiler");
+        // this.field_149936_O = p_149651_1_.registerIcon(this.field_149932_b ?
```

Comment:
The changes to every comment should ideally be removed, because that's just an annoying amount of changed files.